README.md

```diff
@@ -96,110 +77,34 @@ The ready-made commands you can start working with (both on **Windows** and **Li
 
 * `build.cmd all` - runs the following commands: `NBench`, `Tests`, and `Nuget`.
 * `build.cmd compile` - compiles the solution in `Release` mode. The default mode is `Release`, to compile in `Debug` mode => `--configuration debug`
-* `build.cmd tests` - compiles the solution in `Release` mode and runs the unit test suite (all projects that end with the `.Tests.csproj` suffix). All of the output will be published to the `./TestResults` folder.
+* `build.cmd runtests` - compiles the solution in `Release` mode and runs the unit test suite (all projects that end with the `.Tests.csproj` suffix). All of the output will be published to the `./TestResults` folder.
 * `build.cmd nbench` - compiles the solution in `Release` mode and runs the [NBench](https://nbench.io/) performance test suite (all projects that end with the `.Tests.Performance.csproj` suffix). All of the output will be published to the `./PerfResults` folder.
 * `build.cmd nuget` - compiles the solution in `Release` mode and creates Nuget packages from any project that does not have `<IsPackable>false</IsPackable>` set and uses the version number from `GitVersion.SemVer`.
```

Comment:
`signpackages` needs to run as a sub-step of `nuget` FYI - and if no one provided signing credentials then this step needs to be safely skipped as part of a normal `build.cmd nuget` command. This is what our old build system did. Is that how this system behaves today?